### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/tcpdump/actions/runs/5278571637#summary-14288652640

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/tcpdump/actions/runs/5278571945#summary-14288653168

### CODE REVIEW

File: addrtoname.c [hunk 1]
```diff
@@ -943,12 +943,12 @@ static const struct etherlist {
  * and etheraddr_string() fills in the table on demand.  If it doesn't,
  * then we suck in the entire /etc/ethers file at startup.  The idea
  * is that parsing the local file will be fast, but spinning through
- * all the ethers entries via NIS & next_etherent might be very slow.
+ * all the ethers entries via NIS & next_ethernet might be very slow.
```
Comment (jsoref) [inline addrtoname.c:946]: These changes are dependent on https://github.com/the-tcpdump-group/libpcap/pull/1198/commits/949aff552716fa5fe94dc10a1cb2bf9134bcacde in https://github.com/the-tcpdump-group/libpcap/pull/1198 and can be split out / dropped.

I'm going to drop them and address them separately.

File: mib.h [hunk 2]
```diff
@@ -765,13 +765,13 @@ _icmpInParmProbs_obj = {
 	"icmpInParmProbs", 5, 0,
 	NULL, &_icmpInTimeExcds_obj
 },
-_icmpInSrcQuenchs_obj = {
-	"icmpInSrcQuenchs", 6, 0,
+_icmpInSrcQuenches_obj = {
+	"icmpInSrcQuenches", 6, 0,
```
Comment (jsoref) [inline mib.h:769]: Happy to drop, I understand that individually each object is an `ICMP Source Quench`, but the standard English plural would be to use `-es` not `-s`.

File: print-802_11.c [hunk 3]
```diff
@@ -1724,7 +1724,7 @@ static const struct tok category_str[] = {
 	{ 3,   "BA"                  },
 	{ 7,   "HT"                  },
 	{ 13,  "MeshAction"          },
-	{ 14,  "MultiohopAction"     },
+	{ 14,  "MultihopAction"      },
```
Comment (jsoref) [inline print-802_11.c:1727]: hopefully?

File: print-802_15_4.c [hunk 4]
```diff
@@ -250,7 +250,7 @@ static const char *p_mlme_short_names[] = {
 	"TVWS PHY Operating Mode Description IE",	/* 0x2b */
 	"TVWS Device Capabilities IE",			/* 0x2c */
 	"TVWS Device Category IE",			/* 0x2d */
-	"TVWS Device Identiication IE",			/* 0x2e */
+	"TVWS Device Identification IE",			/* 0x2e */
```
Comment (jsoref) [inline print-802_15_4.c:253]: Hopefully?

Comment (jsoref) [inline print-802_15_4.c:253]: I'll fix the whitespace

File: print-802_15_4.c [hunk 5]
```diff
@@ -418,7 +418,7 @@ static const char *mac_c_names[] = {
 #define FC_ADDRESSING_MODE_LONG         0x03
 
 /*
- * IEEE 802.15.4 CRC 16 function. This is using CCITT polynomical of 0x1021,
+ * IEEE 802.15.4 CRC 16 function. This is using the CCITT polynomial of 0x1021,
```
Comment (jsoref) [inline print-802_15_4.c:421]: maybe?

File: print-lisp.c [hunk 6]
```diff
@@ -134,7 +134,7 @@ static const struct tok lisp_type [] = {
 	{ 2, "LISP-Map-Reply"			},
 	{ 3, "LISP-Map-Register"		},
 	{ 4, "LISP-Map-Notify"			},
-	{ 8, "LISP-Encapsulated-Contol-Message" },
+	{ 8, "LISP-Encapsulated-Control-Message" },
```
Comment (jsoref) [inline print-lisp.c:137]: I'll fix up the alignment

File: print-radius.c [hunk 7]
```diff
@@ -132,8 +132,8 @@ static const struct tok radius_command_values[] = {
     { RADCMD_ACCESS_REQ, "Access-Request" },
     { RADCMD_ACCESS_ACC, "Access-Accept" },
     { RADCMD_ACCESS_REJ, "Access-Reject" },
-    { RADCMD_ACCOUN_REQ, "Accounting-Request" },
-    { RADCMD_ACCOUN_RES, "Accounting-Response" },
+    { RADCMD_ACCOUNT_REQ, "Accounting-Request" },
+    { RADCMD_ACCOUNT_RES, "Accounting-Response" },
```
Comment (jsoref) [inline print-radius.c:136]: I'll fix the alignment

File: tcpdump.c [hunk 8]
```diff
@@ -796,7 +796,7 @@ droproot(const char *username, const char *chroot_dir)
 	} else
 		error("Couldn't find user '%.32s'", username);
 #ifdef HAVE_LIBCAP_NG
-	/* We don't need CAP_SETUID, CAP_SETGID and CAP_SYS_CHROOT any more. */
```
Comment (DimitriPapadopoulos) [inline tcpdump.c:799]: It looks like `any more` is correct British English:
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/definition/english/any-more)
* [Cambridge](https://dictionary.cambridge.org/fr/dictionnaire/anglais/any-more)
* [Collins](https://www.collinsdictionary.com/dictionary/english/any-more)

Comment (jsoref) [reply tcpdump.c:799]: Yes, but this repository and its sibling (see https://github.com/the-tcpdump-group/libpcap/pull/1198) tend to be predominantly en-US.

File: print-802_11.c (hunk 3 above)
Comment (fxlb) [reply print-802_11.c:1727]: Yes.

File: print-802_15_4.c (hunk 4 above)
Comment (fxlb) [reply print-802_15_4.c]: Yes.

File: print-802_15_4.c (hunk 5 above)
Comment (fxlb) [reply print-802_15_4.c:421]: Yes.

File: print-radius.c (hunk 7 above)
Comment (fxlb) [reply print-radius.c:136]: Please do it.

File: tcpdump.c (hunk 8 above)
Comment (guyharris) [reply tcpdump.c:799]: "Any more" isn't US English?

For example, Robert Allen Zimmerman, born in Duluth, Minnesota, is generally considered to be from the US, and, [from something he wrote](https://www.bobdylan.com/songs/knockin-heavens-door/):

> Mama, take this badge off of me
> I can’t use it anymore

Comment (guyharris) [reply tcpdump.c:799]: And if the issue is "any more" vs. "anymore", [the Merriam-Webster Dictionary, a US English dictionary, says](https://www.merriam-webster.com/dictionary/anymore):

> Although both *anymore* and *any more* are found in written use, in current writing *anymore* is the more common styling.

so both are valid, although "anymore" is more common currently.

Comment (infrastation) [reply tcpdump.c:799]: That's how I usually split this hair:
* Would you like any more sugar?
* I do not eat sugar anymore.

Comment (fxlb) [reply tcpdump.c:799]: determiner/adverb:
https://dictionary.cambridge.org/fr/grammaire/grammaire-britannique/any-more-or-anymore

**Comment by fxlb** [comment]
If everyone agrees, it's time for a squash before a merge.

**Review by fxlb — APPROVED** [review]

**Comment by infrastation** [comment]
No objections.

**Comment by fxlb** [comment]
Last call for comments...

**Comment by fxlb** [comment]
Thank you!

**Pull request merged**